tests/reporting/test_dispute_bond_token.py

```diff
@@ -20,9 +20,9 @@
 MARKET_TYPE_CATEGORICAL = 0
 MARKET_TYPE_SCALAR = 1
 
-CATEGORICAL_OUTCOME_A = [3, 0, 0]
-CATEGORICAL_OUTCOME_B = [0, 3, 0]
-CATEGORICAL_OUTCOME_C = [0, 0, 3]
+CATEGORICAL_OUTCOME_A = [3*10**17, 0, 0]
```

Comment:
This shouldn't be 3... in fact, 10^18 isn't divisible by 3 so a 3-outcome categorical market cannot have 10^18 as a base (or 10^anything).  This was the thing Joey mentioned about how tick sizes for odd numbered markets will be "odd".

Reply:
The payout numerator here is 3*10**17, which is what the marketDenominator now is for the categorical market. I'm not sure why 10^18 is involved?